gcp_variant_transforms/libs/vcf_file_composer.py

```diff
@@ -179,6 +179,8 @@ def _compose_blobs_to_one(self, blob_prefix):
     """
     blobs_to_be_composed = list(self._bucket.list_blobs(prefix=blob_prefix))
     logging.info('Total number of blobs is %d.', len(blobs_to_be_composed))
+    if not blobs_to_be_composed:
```

Comment:
This would have been a good candidate for a separate commit.  This is part of the value of keeping your commit messages complete from the start: you can then rebase and push when you have multiple unrelated changes in a single PR.

Reply:
I see! I will drop this PR and create a new one. :)